packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html

```diff
@@ -324,7 +388,8 @@
                                                 data-model="option.choosedValue"
                                                 min="0"
                                                 max="option.optional"
-                                                data-on-change="$ctrl.constructor.updateOfferDisplayedPrice(modelValue, offer, option.name, $row)"
+                                                data-on-change="$ctrl.constructor.updateOfferDisplayedPrice(modelValue, offer, option.name, $row); $ctrl.constructor.updateSelectedVoipLineOption(modelValue, offer, option)"
```

Comment:
**nitpick:** Use a single function and call the required functions inside.

Reply:
done